### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by ksemina61**

### CODE REVIEW

**Review by fortvivlan — COMMENTED** [review]
`v2.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225])`
Вообще говоря, эти числа надо оценивать самим по обучающей выборке. Можно оставить ImageNet'овские, если ваши данные похожи на реальные фотографии (https://stackoverflow.com/a/60179019). В вашем случае, вероятно, сработает, однако сомневаюсь, что вы сделали это осознанно, так что убедитесь еще раз, что вы понимаете, что такое нормализация каналов картинок и для чего она нужна.

---

`val_loader = DataLoader(test_dataset, batch_size = 64, shuffle = False)`
Так тест или все же валидация? Тоже подумайте в чем разница между ними

---

```
        pred = self.conv_1(x)
        pred = self.relu(pred)
        pred = self.pool(pred)
        pred = self.conv_2(pred)
        pred = self.relu(pred)
        pred = self.pool(pred)
        pred = self.flatten(pred)
        pred = self.fc_1(pred)
        pred = self.relu(pred)
        pred = self.fc_2(pred)
```
Есть просто nn.Sequential

---

Попытка 2 - вот почему бы не поучить подольше? Видно же, что лосс на валидации стабильно убывает. Может за 100 эпох и выучилось бы что-то хорошее. Еще можно было бы с шагом обучения поэкспериментировать.
 
---

> Все, я сдаюсь. Добавила дата аугментацию

Она и так была все это время. Здесь RandomResizedCrop и RandomHorizontalFlip - аугменатция данных:

```
transform = v2.Compose([
    v2.RandomResizedCrop(size=(224, 224), antialias=True), # antialias=True - сглаживание при изменении размера изображения
    # я думала, что обрезка и изменение размера помогут мне создать вариации исходных изображений, что является одной из форм увеличения объема данных
    v2.RandomHorizontalFlip(p=0.5), # Горизонтальное переключение
    v2.ToDtype(torch.float32, scale=True), # scale=True - масштабируются от диапазона [0, 255] до диапазона [0, 1] путем деления на 255
    v2.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225]),
])
```
Однако на тесте ее нужно выключать, а у вас и на тесте происходит рандомный флип, что усложняет модели задачу во время инференса.
